third_party/2and3/click/decorators.pyi

```diff
@@ -83,7 +83,8 @@ def argument(
     metavar: Optional[str] = ...,
     expose_value: bool = ...,
     is_eager: bool = ...,
-    envvar: Optional[Union[str, List[str]]] = ...
+    envvar: Optional[Union[str, List[str]]] = ...,
+    autocompletion: Callable[[Any, List[str]], str], List[Union[str, Tuple[str, str]]] = ...,
```

Comment:
This should be `Optional` as it defaults to `None` in `Parameter.__init__()`. Also, it seems the brackets are misaligned, which is also why CI fails.